### ISSUE TITLE
Can I please ask a few beginner questions?

### ISSUE THREAD

**Issue by chrisjdixon** [teticio/lambda-scraper#1]
Hi @teticio , I've come across [your guide](https://medium.com/nerd-for-tech/web-scraping-with-a-proxy-pool-the-cheap-way-4c7d6fc9f859) for this repo. Looks amazing! I'm trying to follow instructions but have never done anything with Terraform, AWS CLI, or Lambda before so I'm a little confused about a few things. Your instructions seem clear and straightforward but I'm a little lost through inexperience.

Do you mind if I ask a few dumb questions? I don't know where else I could message you so is this the most appropriate place? In the issues section of your repo? Because I wouldn't classify this as an "issue" 😅

**Comment by teticio** [comment]
Sure. You can email me at ***@***.*** Happy to help

**Comment by ElMurte** [comment]
![image](https://user-images.githubusercontent.com/32709092/127399716-26503e6d-b9e6-4a21-999c-f3e5e24be5ce.png)
Hi man, i'm a beginner in that field, i followed your instructions but when i lunch i got the error in the image

`PS C:\Users\elvis\Desktop\Web_scrapper\lambda-scraper> terraform apply -auto-approve
╷
│ Error: Plugin error
│
│   with module.lambda_function[3].module.source_zip_file.data.external.archive[0],
│   on .terraform\modules\lambda_function.source_zip_file\main.tf line 28, in data "external" "archive":
│   28: data "external" "archive" {
│
│ The plugin returned an unexpected error from plugin.(*GRPCProvider).ReadDataSource: rpc error: code = Internal desc =
│ grpc: error while marshaling: string field contains invalid UTF-8`

**Comment by teticio** [comment]
What OS are you running on? I just tried running it on Windows and had some issues (different from yours). Unfortunately it looks like an issue with the Lambda function module which is developed by someone else. It works on Linux no problem.

I did remove a file from the repo that might solve your problems. Try running everything from scratch in a fresh directory.

**Comment by teticio** [comment]
OK, I have changed the module for Lambda functions to another provider and it now works on Windows. There was a small wrinkle when I ran it - it had some problem moving a file as it was in use - but running it twice did teh trick (don't worry, terraform will not create double the resources if you run it twice!). Hope this helps. (Again start from scratch in a new folder. To be on the safe side, run "terraform apply -destroy" in your existing folder in case there are some artefacts left over from a partial deployment.)
